app/Config/Schema/db_acl.php

```diff
@@ -12,91 +12,92 @@
  * Redistributions of files must retain the above copyright notice.
  *
  * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
  * @link          http://cakephp.org CakePHP(tm) Project
- * @package       app.Config.Schema
  * @since         CakePHP(tm) v 0.2.9
+ *
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
 
 /**
  * Using the Schema command line utility
- * cake schema run create DbAcl
- *
+ * cake schema run create DbAcl.
  */
-class DbAclSchema extends CakeSchema {
+class DbAclSchema extends CakeSchema
+{
+    /**
+     * Before event.
+     *
+     * @param array $event The event data.
+     *
+     * @return bool Success
+     */
+    public function before($event = array())
+    {
+        return true;
+    }
 
-/**
- * Before event.
- *
- * @param array $event The event data.
- * @return bool Success
- */
-	public function before($event = array()) {
-		return true;
-	}
+    /**
+     * After event.
+     *
+     * @param array $event The event data.
+     */
+    public function after($event = array())
+    {
+    }
 
-/**
- * After event.
- *
- * @param array $event The event data.
- * @return void
- */
-	public function after($event = array()) {
-	}
+    /**
+     * ACO - Access Control Object - Something that is wanted.
+     */
+    public $acos = array(
+        'id' => array('type' => 'integer', 'null' => false, 'default' => null, 'length' => 10, 'key' => 'primary'),
+        'parent_id' => array('type' => 'integer', 'null' => true, 'default' => null, 'length' => 10),
+        'model' => array('type' => 'string', 'null' => true),
+        'foreign_key' => array('type' => 'integer', 'null' => true, 'default' => null, 'length' => 10),
+        'alias' => array('type' => 'string', 'null' => true),
+        'lft' => array('type' => 'integer', 'null' => true, 'default' => null, 'length' => 10),
+        'rght' => array('type' => 'integer', 'null' => true, 'default' => null, 'length' => 10),
+        'indexes' => array(
+            'PRIMARY' => array('column' => 'id', 'unique' => 1),
+            'idx_acos_lft_rght' => array('column' => array('lft', 'rght'), 'unique' => 0),
+            'idx_acos_alias' => array('column' => 'alias', 'unique' => 0),
```

Comment:
__[staging][code_sniffer]__
* Array key not aligned correctly; expected 22 spaces but found 12
* Array with multiple values cannot be declared on a single line